### PR TITLE
Updatating the DNS examples to non tracking

### PR DESCRIPTION
**Pull request by spirillen**
As the DNS servers added in this document refers to well known tracking servers, I've taken the libaty to replace them to some DNS servers that should not be tracking servers.

In this case quad9's and yours

-------
Known non-tracking DNS servers that at the same time offers protection from tracking, spyware, adware, phishing etc.:

These are using Ultimate.Hosts.Blacklist lists for blacklisting <https://github.com/Ultimate-Hosts-Blacklist/Ultimate.Hosts.Blacklist/issues/590>
  - 88.198.70.38
  - 2a01:4f8:140:5021::38

  - 88.198.70.39
  - 2a01:4f8:140:5021::39
-------
These are using <https://www.mypdns.org/> RPZ zones <https://www.mypdns.org/source/Matrix/repository/master/> 

  - 95.216.209.53
  - 2a01:4f8:1c0c:5f61::53

  - 116.203.32.67
  - 2a01:4f9:c010:410e::53
-------

### CODE REVIEW

**Comment by spirillen** [comment]
Awaiting the review and comments on <https://github.com/spirillen/PyFunceble/pull/9>

File: docs/components/databases.rst [hunk 1]
```diff
@@ -6,6 +6,7 @@ Why do we use "databases"?
 
 We use databases to store data while we run the tests. When globally talking
 about databases, we are indirectly talking about the following subsystems.
+(JSON)
```
Comment (funilrys) [inline docs/components/databases.rst:9]: Where does this come from?

File: docs/components/databases.rst [hunk 2]
```diff
@@ -102,10 +107,71 @@ How to use the :code:`mysql` or :code:`mariadb` format?
...
+   :code:`mariadb` or :code:`mysql`.
 5. Play with PyFunceble!
 
 .. note::
-    If the environment variables are not found, you will be asked to prompt the
-    information.
+    If the environment variables are not found, you will be asked to
+    prompt the information.
+
+SQL Layout:
```
Comment (funilrys) [inline docs/components/databases.rst:118]: As of a possible merge of https://github.com/funilrys/PyFunceble/tree/issue-118 in the coming days, All INT (except for `http_status_code`) will be converted to BIGINT.

File: docs/facts/known_issues.rst [hunk 3]
```diff
@@ -30,3 +30,26 @@ However, the right way to do this is by first locate where your's
 
 :code:`find /usr/lib/ -type f -iname 'libffi.so.*'` and then apply the
 softlink to :code:`libffi.so.7`
+
+Urllib3
```
Comment (funilrys) [inline docs/facts/known_issues.rst:34]: I'd likely put this section on hold as (and sorry for misleading this, I don't know where my mind was!) the `pyfunceble-not-resolved` domain is a PyFunceble reserved/generated host which is used when the IP was not found while trying to resolve.

File: docs/components/databases.rst (hunk 1 above)
Comment (spirillen) [reply docs/components/databases.rst:9]: Line break. (The 80 chars for consoles support)

I added it as this part is about JSON and does have nothing to do with the SQL layout

File: docs/components/databases.rst (hunk 2 above)
Comment (spirillen) [reply docs/components/databases.rst:118]: I'll take note of that.

What about my question marks in the `SQL Layout:` L118-L177 ?

Another thing I just noticed, it the lines L73-L108 even relevant anymore after changing to sqlachemy?

File: docs/facts/known_issues.rst (hunk 3 above)
Comment (spirillen) [reply docs/facts/known_issues.rst]: Removed by <https://github.com/funilrys/PyFunceble/pull/122/files#diff-5c443d7148beb3d2f8a2ccb24c6a7607>

File: docs/components/databases.rst (hunk 2 above)
Comment (spirillen) [reply docs/components/databases.rst:118]: Let's get this one merged 

Then we polish it on the dev branch

**Comment by spirillen** [comment]
Let's get this one merged 

Then we polish it on the dev branch

**Comment by funilrys** [comment]
Closing. Please resubmit to the `4.0.0-dev` branch.

**Comment by spirillen** [comment]
Seem right, as the table have been under a heavy transformation as well

**Pull request closed without merging**